source/tutorials/php.rst

```diff
@@ -0,0 +1,128 @@
+.. _php:
+
+Installing and Configuring PHP
+******************************
+
+This tutorial describes how to configure and use PHP and PHP extensions on |CL-ATTR|. Although we specifically address PHP this tutorial can serve as a general guide for working with applications in the |CL| :ref:`stateless` environment.
+
+We will focus on enabling PHP and PHP extensions in |CL|.  For a complete :abbr:`LAMP (Linux, Apache, MySQL, PHP)` setup in |CL| refer to the :ref:`web-server-install` section of the :ref:`wordpress` tutorial. For this tutorial you do not need to install MariaDB or WordPress.
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Prerequisites
+*************
+
+* :ref:`Install <bare-metal-install-desktop>` |CL| on your host system
+* Follow :ref:`web-server-install` to install Apache\* and PHP
+
+
+Configuring PHP
+***************
+
+.. important::
+
+   This tutorial does not cover configuring the php-fpm service. There is a difference between php and php-fpm here. php-fpm sets configuration for the process management and pools of worker bees, and can optionally set some of the values defined in the :file:`php.ini` file. This tutorial is only looking at the PHP configuration directives, not those of php-fpm.
+
+
+By default PHP looks for configuration settings in the :file:`php.ini` file, which resides in the `usr/share/defaults/php/` path. Because |CL| is designed to be a :ref:`stateless` operating system, you must create an optional configuration file to override the default values. Every time :command:`swupd` updates the system it will overwrite changes to the `/usr/share/defaults` file structure.
```

Comment:
```suggestion
By default PHP looks for configuration settings in the :file:`php.ini` file, which resides in the :file:`usr/share/defaults/php/` path. Because |CL| is designed to be a :ref:`stateless` operating system, you must create an optional configuration file to override the default values. Every time :command:`swupd` updates the system it will overwrite changes to the :file:`/usr/share/defaults` file structure.
```

`:file:` syntax for paths just to make it render better